example-apps/dashnote-starter/src/styles.css

```
@@ -0,0 +1,237 @@
+  margin: 0;
+  font-size: 1rem;
+  font-weight: 600;
+}
+
+.note-message {
+  margin: 0.25rem 0 0;
+  max-width: 70ch;
+  color: #374151;
+  font-size: 0.9rem;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)